packages/firestore/test/unit/local/indexeddb_persistence.test.ts

```diff
@@ -77,8 +78,9 @@ function withDb(
     });
 }
 
-async function withPersistence(
+async function withConfiguredPersistence(
```

Comment:
Nit: Maybe withPersistenceInternal/withCustomPersistence? I am not a big fan of these names either, but just throwing it out there since all of our persistence implementations are configured.

Reply:
I guess I'll do `withCustomPersistence`.